prep-javascript-dom-events/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Events</title>
+</head>
+
+<body>
+  <button class="hello-world-btn">Hello World!</button>
+  <script>
+    document.addEventListener('DOMContentLoaded', console.log('content loaded!'));
+    window.addEventListener('mousemove', console.log('mouse is moving!'));
+    var $helloWorld = document.querySelector(button.hello - world - btn);
+    $helloWorld.addEventListener('click', console.log('hello world!'));
```

Comment:
Hi Nadia, for each event listener, we want to define a function and put the console log in its code block. That way the console log doesn't happen until the browser fires the event.

```js
var $button = document.querySelector('button');
$button.addEventListener('click', function () {
  console.log('a click happened'); // won't run until the button is actually clicked.
})
```